cohort-engine/src/main/java/com/ibm/cohort/engine/MultiFormatLibrarySourceProvider.java

```diff
@@ -37,10 +42,10 @@
 	 * @param sourceFormat      Source format of the stream of interest
 	 * @return InputStream where source data can be read.
```

Comment:
I noticed some of the javadocs still call out `InputStream` where the return type has changed.

Reply:
Whoops. Fixed!